### PR TITLE
Add support for reviung53

### PR DESCRIPTION
**Pull request by Dani-Kirby**
This PR adds a Reviung53 vial directory and keymap files.

### CODE REVIEW

File: keyboards/reviung/reviung53/keymaps/vial/rules.mk [hunk 1]
```diff
@@ -0,0 +1,5 @@
+VIA_ENABLE = yes
+VIAL_ENABLE = yes
+VIAL_INSECURE = yes
```
Comment (lesshonor) [inline keyboards/reviung/reviung53/keymaps/vial/rules.mk]:
```suggestion
```

`VIAL_INSECURE = yes` is [not permitted in the main `vial-qmk` repository](https://get.vial.today/docs/porting-to-vial.html#6-set-up-a-secure-unlock-combination).

File: keyboards/reviung/reviung53/keymaps/vial/readme.md
Comment (lesshonor) [inline keyboards/reviung/reviung53/keymaps/vial/readme.md:1]: Is this file really necessary?

**Review by lesshonor — CHANGES_REQUESTED** [review]

**Comment by Dani-Kirby** [comment]
Requested changes have been resolved. Please let me know if you need anything else

File: keyboards/reviung/reviung53/readme.md
Comment (lesshonor) [inline keyboards/reviung/reviung53/readme.md:1]: Please restore this file. It was `readme.md` *within the Vial keymap folder* that I asked about. The keyboard-level file actually contains useful information...

File: keyboards/reviung/reviung53/keymaps/vial/config.h
Comment (lesshonor) [inline keyboards/reviung/reviung53/keymaps/vial/config.h:1]: Please review [the documentation I linked earlier](https://get.vial.today/docs/porting-to-vial.html#6-set-up-a-secure-unlock-combination), and remember to test changes locally. Without an unlock combination, this keymap won't compile.

**Review by lesshonor — CHANGES_REQUESTED** [review]

**Comment by Dani-Kirby** [comment]
Requested changes should now be correctly implemented -- Firmware compiles

**Review by lesshonor — APPROVED** [review]

**Pull request merged**